src/text_object.h

```diff
@@ -76,25 +77,37 @@ void gen_print_nothing(struct text_object *, char *, int);
 void gen_print_obj_data_s(struct text_object *, char *, int);
 
 class legacy_cb: public conky::callback<void *, int (*)()> {
-    typedef conky::callback<void *, int (*)()> Base;
+	typedef conky::callback<void *, int (*)()> Base;
```

Comment:
Do you prefer hard tabs or soft tabs? Kate took the liberty of changing these tabs automatically, but I figured I would keep the changes, since most of the files seem to use hard tabs. Sorry for all the issues. I'm making note of style preferences for next time.